bedrock/firefox/templates/firefox/includes/send-to-device.html

```diff
@@ -4,7 +4,7 @@
   file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
 
-<section id="{{ dom_id }}" class="send-to-device {{ class_name }}" data-spinner-color="{{ spinner_color }}" data-testid="send-to-device-form">
+<section id="{{ dom_id }}" class="send-to-device {{ class_name }}" data-testid="send-to-device-form">
```

Comment:
1. the `spinner_color` is no-op elsewhere, I'll address the actual macros at a later point, possibly removing the colour param from the helpers (& docs) altogether…
2. to minimise the diff surface I haven't changed what wasn't necessary in the markup, e.g. I haven't considered adding `role="progressbar"` and/or some form of image-replaced "loading" text for AT, given the timeouts are relatively short so there's not much value when similar interactions elsewhere also don't provide this, and also markup changes would trigger related test updates etc. so just kept it simple.

Reply:
Also, I haven't changed the original logic where the placeholder elements are positioned, so in case of _newsletter management_ it is swapped with error box if present and alternates in the same space; in case of _send to device_ widgets the error overview is placed in the beginning of the form and the spinner at the bottom:

https://github.com/user-attachments/assets/f13a455b-1880-466e-941c-656c61b87828

(which also seem sufficient for the task.)